source/common/assets/less/main.less

```diff
@@ -0,0 +1,14 @@
+// KaTeX
+@import '~katex/dist/katex.css';
+
+// Clarity
+@import '~@clr/icons/clr-icons.css';
+
+// Tippy
+@import '~tippy.js/dist/tippy.css';
+
+// App layout
+@import './geometry/geometry-main.less';
+
+// Main theme
+@import './theme-berlin/theme-main.less';
```

Comment:
Why do you only import the main theme here? I assume this will be resolved once we also speak about the way the theme-switching is currently handled! ;)